pyproject.toml

```diff
@@ -1,5 +1,41 @@
+[project]
+name = "tiledb-cloud"
+description = "TileDB Cloud platform Python client"
+dynamic = ["version"]
+
+dependencies = [
+  "attrs>=21.4.0",
+  "certifi",
+  "cloudpickle==1.4.1",
+  # as of 2021-07-06, the runtime (pinned to 1.1) is incompatible with 1.3
+  "pandas<1.3; python_version < '3.10'",
+  "pandas>=1.3; python_version >= '3.10'",
+  "pyarrow>=3.0.0",
+  "python-dateutil",
+  "six>=1.10",
+  "tiledb>=0.5.0",
+  "urllib3>=1.26",
+]
+
+[project.optional-dependencies]
+viz-tiledb = ["networkx>=2", "pydot", "tiledb-plot-widget>=0.1.7"]
+viz-plotly = ["networkx>=2", "plotly>=4", "pydot"]
+all = ["networkx>=2", "plotly>=4", "pydot", "tiledb-plot-widget>=0.1.7"]
+
+[project.urls]
+homepage = "https://tiledb.com"
+repository = "https://github.com/TileDB-Inc/TileDB-Cloud-Py"
+
+
 [build-system]
 requires = ["setuptools>=42", "wheel", "setuptools_scm>=6"]
 
+[tool.setuptools]
+zip-safe = false
+
+[tool.setuptools.packages.find]
+where = ["src"]
+exclude = ["tiledb"]
```

Comment:
Why the exclude here?

Reply:
I could have sworn this was necessary to make it not clobber `tiledb/__init__.py`, but apparently not? No complaints about removing it from me.